magma_access_gateway_installer/agw_installer.py

```diff
@@ -42,8 +42,8 @@ def install(self):
             self.start_magma()
             logger.info(
                 "Magma AGW deployment completed successfully!\n"
-                "System will now go down for the reboot to apply all changes.\n"
-                "After configuring Magma AGW run magma-access-gateway.post-install to verify "
+                "\t\tSystem will now go down for the reboot to apply all changes.\n"
+                "\t\tAfter configuring Magma AGW run magma-access-gateway.post-install to verify "
                 "configuration."
```

Comment:
Is it really to verify `configuration`? Here we use configuration for when we relate to orc8r right? So maybe to `validate access gateway is installed correctly`.

Reply:
So it does both actually. This action won't succeed unless AGW is fully configured (see `check_cloud_check_in()` in **agw_post_install.py**)